ontoma/EFO_Handler.py

```diff
@@ -0,0 +1,177 @@
+"""Class to handle EFO ontology related functions."""
+
+import logging
+import os
+import urllib.request
+
+import pandas as pd
+import pronto
+import requests
+from retry import retry
+from typing import Dict
+
+from ontoma import ontology
+
+logger = logging.getLogger(__name__)
+
+
+class efo_handler:
```

Comment:
While in rare cases PEP8 does allow this in rare cases, I think in our case it's best to use the mode widespread camelcase convension for this class name (`EFOHandler`)